azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/redux/reducer/AppStateReducer.kt

```diff
@@ -15,11 +16,16 @@ internal class AppStateReducer(
     private val errorReducer: ErrorReducer,
     private val navigationReducer: NavigationReducer,
     private val audioSessionReducer: AudioSessionReducer,
+    private val localOptions: CallCompositeLocalOptions,
```

Comment:
we should not pass local options here. 
Just pass isCameraOn and isMicrophoneOn

Reply:
yes, makes sense. I tried to keep data flow aligned with iOS. we both can refactor I think.